1.2.c

```diff
@@ -0,0 +1,37 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include <stdio.h>
+#include <math.h>
+
+int main()
+{
+	printf ("Input a: ");

```

Comment:
```suggestion
	printf ("Enter a dividend: ");
```
Пользователь программы ничего не знает про `a`. Сообщение должно быть достаточно информативным, чтобы пользователь понимал, что он вводит (и для остальных переменных).